website/react-magma-docs/src/pages/api/theme.mdx

```diff
@@ -55,6 +55,39 @@ const magmaDark = Object.assign({}, magma, {
   }),
 });
 
+const ceslV2 = Object.assign({}, magma, {
+  bodyFont: '"Open Sans",Helvetica,sans-serif',
+  bodyExpressiveFont: '"Open Sans",Helvetica,sans-serif',
+  bodyNarrativeFont: "'Noto Serif',Times New Roman,serif",
+  borderRadius: '4px',
+  colors: Object.assign({}, magma.colors),
+  headingFont: '"Open Sans",Helvetica,sans-serif',
+  headingExpressiveFont: '"Open Sans",Helvetica,sans-serif',
+  headingNarrativeFont: "'Noto Serif',Times New Roman,serif",
+
+  // Typography
+  typeScale: magma.typeScale,
+  typographyExpressiveVisualStyles: {
+    headingXLarge: {
+      mobile: magma.typeScale.size08,
+      desktop: magma.typeScale.size11,
+      fontWeight: 600,
+    },
+  },
+
+  appBar: {
+    height: '56px',
+  },
+
+  tooltip: {
+    backgroundColor: magma.colors.neutral,
+    fontWeight: 600,
+    inverse: {
+      textColor: magma.colors.neutral,
+    },
+  },
+});
+
```

Comment:
I think this example should be even more different to the other two Magma ones, otherwise I'm not sure it adds a lot of value.